canvas_modules/harness/src/client/App.js

```diff
@@ -2192,7 +2192,24 @@ class App extends React.Component {
 					{ divider: true },
 					{ action: "color-subpanel", iconEnabled: (<ColorPalette size={32} />), label: "Color picker", enable: true,
 						subPanel: ColorPicker, subPanelData: { clickActionHandler: (color) => window.alert("Color selected = " + color) } },
-					{ divider: true }
+					{ divider: true },
+					{ action: "save", iconEnabled: (<Save size={32} />), enable: true,
+						tooltip:
+							<div>
+								<br />
+								<p style={ { fontSize: "12px" } } ><strong>jjennings</strong> saved the flow at 8:18AM.</p>
+								<br />
+								<ul>
+									<li><p style={ { fontSize: "12px" } }><strong>Reload</strong> to view changes. Caution: you will lose your changes.</p></li>
+									<li><p style={ { fontSize: "12px" } } ><strong>Save as</strong> to save your changes as a new flow</p></li>
+								</ul>
+								<br />
+								<div style={ { display: "flex", justifyContent: "space-between" } }>
+									<Button kind="secondary" size="sm" style={ { width: "30px", height: "10px", color: "lightblue" } }>Save</Button>
+									<Button kind="danger" size="sm" style={ { width: "30px", height: "10px" } }>Reload</Button>
+								</div>
+							</div>
+					}
```

Comment:
Can you pull out this JSX for the tooltip and assign it to a variable above (like the way subMenuTextSize and subMenuSize are created and then just include the variable here?